### PR TITLE
Feature/synapse service mongodb sample

### PR DESCRIPTION
**Pull request by whoswendy**
This PR includes sample for end to end integration with service-rest and data-mongodb.

### CODE REVIEW

File: service/service-samples/pom.xml [hunk 1]
```diff
@@ -16,6 +16,8 @@
     <modules>
         <module>sample-service-graphql-book</module>
         <module>sample-service-rest-book</module>
+        <module>sample-service-rest-mongodb-book</module>
+<!--        <module>sample-service-rest-native-book</module>-->
```
Comment (guttff) [inline service/service-samples/pom.xml]: remove commented code

**Review by guttff — CHANGES_REQUESTED** [review]

File: service/service-samples/sample-service-rest-mongodb-book/src/test/java/io/americanexpress/service/book/rest/service/helper/ReadBookResponseCreatorTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        Assertions.assertNotNull(response);
+    }
+
+    @Test
+    void create_givenNull_expectedNull() {
+        ReadBookResponse response = ReadBookResponseCreator.create(null);
+        Assertions.assertNull(response);
+    }
+
+
```
Comment (guttff) [inline service/service-samples/sample-service-rest-mongodb-book/src/test/java/io/americanexpress/service/book/rest/service/helper/ReadBookResponseCreatorTest.java]: remove extra line

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/ReactiveDeleteBookService.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+public class ReactiveDeleteBookService extends BaseDeleteReactiveService {
+
+    private final BookRepository bookRepository;
+
+    public ReactiveDeleteBookService(BookRepository bookRepository) {
+        this.bookRepository = bookRepository;
+    }
+
+    @Override
+    protected Mono<Void> executeDelete(String identifier) {
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/ReactiveDeleteBookService.java]: I know we have identifier as the default naming of the string. Can we instead call this String "title."

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/controller/DeleteBookReactiveController.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package io.americanexpress.service.book.rest.controller;
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/controller/DeleteBookReactiveController.java]: Add Copyright

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/controller/UpdateBookReactiveController.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
+package io.americanexpress.service.book.rest.controller;
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/controller/UpdateBookReactiveController.java]: Add Copyright

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/CreateBookRequest.java [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    private String title;
+
+    @NotBlank
+    private String author;
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/CreateBookRequest.java]: add java docs on getter & setter

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/ReadBookRequest.java [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+/**
+ * {@code ReadBookRequest} is the request for the read book operation.
+ */
+public class ReadBookRequest implements BaseServiceRequest {
+
+    @NotBlank
+    private String title;
+
+    public String getTitle() {
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/ReadBookRequest.java]: add java doc on getter & setter

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/ReadBookResponse.java [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+    private String title;
+
+    private String author;
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/ReadBookResponse.java]: add javadoc on getter & setter

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/UpdateBookRequest.java [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+    @NotBlank
+    private String title;
+
+    @NotBlank
+    private String author;
+
+    @Min(0)
+    private int numberOfCopies;
+
+    public String getTitle() {
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/UpdateBookRequest.java]: Add javadoc on getter & setter

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/CreateBookReactiveService.java [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+public class CreateBookReactiveService extends BaseCreateReactiveService<CreateBookRequest, CreateBookResponse> {
+
+    private final BookRepository bookRepository;
+
+    public CreateBookReactiveService(BookRepository bookRepository) {
+        this.bookRepository = bookRepository;
+    }
+
+    @Override
+    protected Mono<CreateBookResponse> executeCreate(CreateBookRequest request) {
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/CreateBookReactiveService.java]: Add javadoc on method

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/DeleteBookReactiveService.java [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+public class DeleteBookReactiveService extends BaseDeleteReactiveService {
+
+    private final BookRepository bookRepository;
+
+    public DeleteBookReactiveService(BookRepository bookRepository) {
+        this.bookRepository = bookRepository;
+    }
+
+    @Override
+    protected Mono<Void> executeDelete(String title) {
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/DeleteBookReactiveService.java]: Add javadoc

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/ReadPolyBookReactiveService.java [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+public class ReadPolyBookReactiveService extends BaseReadPolyReactiveService<ReadBookRequest, ReadBookResponse> {
+
+    private final BookRepository bookRepository;
+
+    public ReadPolyBookReactiveService(BookRepository bookRepository) {
+        this.bookRepository = bookRepository;
+    }
+
+    @Override
+    protected Flux<ReadBookResponse> executeRead(ReadBookRequest request) {
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/ReadPolyBookReactiveService.java]: Add javadoc

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/UpdateBookReactiveService.java [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+public class UpdateBookReactiveService extends BaseUpdateReactiveService<UpdateBookRequest> {
+
+    private final BookRepository bookRepository;
+
+    public UpdateBookReactiveService(BookRepository bookRepository) {
+        this.bookRepository = bookRepository;
+    }
+
+    @Override
+    protected Mono<Void> executeUpdate(UpdateBookRequest request) {
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/UpdateBookReactiveService.java]: Add javadoc

File: data/data-samples/sample-data-mongodb-reactive/src/main/java/io/americanexpress/data/book/entity/BookEntity.java [hunk 14]
```diff
@@ -41,4 +43,12 @@ public String getAuthor() {
     public void setAuthor(String author) {
         this.author = author;
     }
+
```
Comment (andCastStatic) [inline data/data-samples/sample-data-mongodb-reactive/src/main/java/io/americanexpress/data/book/entity/BookEntity.java:75]: add javadoc to getter & setters

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/config/BookConfig.java [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+/**
+ * {@code BookConfig} is the configuration class for the Book Application.
+ */
+@Configuration
+@PropertySource("classpath:/service-book-application.properties")
+@ComponentScan(basePackages = "io.americanexpress.service.book.rest")
+@Import({ServiceRestConfig.class, BookDataConfig.class})
+public class BookConfig {
+
+    public BookConfig() {}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/config/BookConfig.java]: is this constructor needed?

File: service/service-samples/sample-service-rest-mongodb-book/src/main/resources/service-book-application.properties [hunk 16]
```diff
@@ -0,0 +1,18 @@
...
+#* Unless required by applicable law or agreed to in writing, software distributed under the License
+#* is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#* or implied. See the License for the specific language governing permissions and limitations under
+#* the License.
+#*/
+spring.application.name=Book Rest Service
+spring.main.allow-bean-definition-overriding=true
+server.port=8080
+wavefront.application.name=bookstore
+wavefront.application.service=example-service-rest-book
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/resources/service-book-application.properties]: add line eof

File: data/data-samples/sample-data-mongodb-reactive/src/main/java/io/americanexpress/data/book/entity/BookEntity.java [hunk 17]
```diff
@@ -41,4 +43,12 @@ public String getAuthor() {
...
+    public int getNumberOfCopies() {
+        return numberOfCopies;
+    }
+
+    public void setNumberOfCopies(int numberOfCopies) {
+        this.numberOfCopies = numberOfCopies;
+    }
```
Comment (andCastStatic) [inline data/data-samples/sample-data-mongodb-reactive/src/main/java/io/americanexpress/data/book/entity/BookEntity.java:92]: EOF line needed

File: data/data-samples/sample-data-mongodb-reactive/src/main/java/io/americanexpress/data/book/repository/BookRepository.java [hunk 18]
```diff
@@ -25,4 +26,8 @@
 public interface BookRepository extends ReactiveMongoRepository<BookEntity, String> {
 
     Flux<BookEntity> findByTitle(String title);
+
+    Mono<BookEntity> findByTitleAndAuthor(String title, String author);
+
+    Mono<Void> deleteByTitle(String title);
 }
```
Comment (andCastStatic) [inline data/data-samples/sample-data-mongodb-reactive/src/main/java/io/americanexpress/data/book/repository/BookRepository.java:34]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/BookApplication.java [hunk 19]
```diff
@@ -0,0 +1,45 @@
...
+     * Main method to run the Spring Boot Book Application.
+     *
+     * @param args the args
+     */
+    public static void main(String[] args) {
+        SpringApplication.run(BookApplication.class, args);
+        LOGGER.info("Rest Book Application is up and running ...");
+    }
+
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/BookApplication.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/config/BookConfig.java [hunk 20]
```diff
@@ -0,0 +1,34 @@
...
+
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/config/BookConfig.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/config/BookEndpoints.java [hunk 21]
```diff
@@ -0,0 +1,25 @@
...
+/**
+ * {@code BookEndpoints} contains the endpoint for the Book API.
+ */
+public class BookEndpoints {
+
+    /**
+     * The constant BOOK_ENDPOINT.
+     */
+    public static final String BOOK_ENDPOINT = "/v1/books";
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/config/BookEndpoints.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/controller/CreateBookReactiveController.java [hunk 22]
```diff
@@ -0,0 +1,30 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * {@code CreateBookReactiveController} creates a book given a CreateBookRequest.
+ */
+@RestController
+@RequestMapping(BookEndpoints.BOOK_ENDPOINT)
+public class CreateBookReactiveController extends BaseCreateReactiveController<CreateBookRequest, CreateBookResponse, CreateBookReactiveService> {
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/controller/CreateBookReactiveController.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/controller/DeleteBookReactiveController.java [hunk 23]
```diff
@@ -0,0 +1,15 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * {@code DeleteBookReactiveController} deletes a book given the title of the book.
+ */
+@RestController
+@RequestMapping(BookEndpoints.BOOK_ENDPOINT)
+public class DeleteBookReactiveController extends BaseDeleteReactiveController<DeleteBookReactiveService> {
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/controller/DeleteBookReactiveController.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/controller/ReadPolyBookReactiveController.java [hunk 24]
```diff
@@ -0,0 +1,30 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * {@code ReadPolyBookReactiveController} retrieves books given a ReadBookRequest.
+ */
+@RestController
+@RequestMapping(BookEndpoints.BOOK_ENDPOINT)
+public class ReadPolyBookReactiveController extends BaseReadPolyReactiveController<ReadBookRequest, ReadBookResponse, ReadPolyBookReactiveService> {
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/controller/ReadPolyBookReactiveController.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/controller/UpdateBookReactiveController.java [hunk 25]
```diff
@@ -0,0 +1,29 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * {@code UpdateBookReactiveController} updates a book given an UpdateBookRequest.
+ */
+@RestController
+@RequestMapping(BookEndpoints.BOOK_ENDPOINT)
+public class UpdateBookReactiveController extends BaseUpdateReactiveController<UpdateBookRequest, UpdateBookReactiveService> {
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/controller/UpdateBookReactiveController.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/CreateBookRequest.java [hunk 26]
```diff
@@ -0,0 +1,72 @@
...
+
+    /**
+     * Sets author.
+     *
+     * @param author the author
+     */
+    public void setAuthor(String author) {
+        this.author = author;
+    }
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/CreateBookRequest.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/CreateBookResponse.java [hunk 27]
```diff
@@ -0,0 +1,22 @@
...
+ */
+package io.americanexpress.service.book.rest.model;
+
+import io.americanexpress.synapse.service.rest.model.BaseServiceResponse;
+
+/**
+ * {@code CreateBookResponse} is the response for the create book operation.
+ */
+public class CreateBookResponse extends BaseServiceResponse {
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/CreateBookResponse.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/ReadBookRequest.java [hunk 28]
```diff
@@ -0,0 +1,48 @@
...
+
+    /**
+     * Sets title.
+     *
+     * @param title the title
+     */
+    public void setTitle(String title) {
+        this.title = title;
+    }
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/ReadBookRequest.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/ReadBookResponse.java [hunk 29]
```diff
@@ -0,0 +1,91 @@
...
+
+    /**
+     * Sets number of copies.
+     *
+     * @param numberOfCopies the number of copies
+     */
+    public void setNumberOfCopies(int numberOfCopies) {
+        this.numberOfCopies = numberOfCopies;
+    }
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/ReadBookResponse.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/UpdateBookRequest.java [hunk 30]
```diff
@@ -0,0 +1,88 @@
...
+
+    /**
+     * Sets number of copies.
+     *
+     * @param numberOfCopies the number of copies
+     */
+    public void setNumberOfCopies(int numberOfCopies) {
+        this.numberOfCopies = numberOfCopies;
+    }
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/model/UpdateBookRequest.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/CreateBookReactiveService.java [hunk 31]
```diff
@@ -0,0 +1,58 @@
...
+        BookEntity bookEntity = new BookEntity();
+        bookEntity.setTitle(request.getTitle());
+        bookEntity.setAuthor(request.getAuthor());
+
+        return bookRepository.save(bookEntity).map(book -> new CreateBookResponse()).switchIfEmpty(Mono.empty());
+
+    }
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/CreateBookReactiveService.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/DeleteBookReactiveService.java [hunk 32]
```diff
@@ -0,0 +1,50 @@
...
+    /**
+     * Deletes book from database.
+     *
+     * @param title the title of book to be deleted
+     */
+    @Override
+    protected Mono<Void> executeDelete(String title) {
+        return bookRepository.deleteByTitle(title);
+    }
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/DeleteBookReactiveService.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/ReadPolyBookReactiveService.java [hunk 33]
```diff
@@ -0,0 +1,57 @@
...
+     * @param request the readBookRequest
+     */
+    @Override
+    protected Flux<ReadBookResponse> executeRead(ReadBookRequest request) {
+        return bookRepository.findByTitle(request.getTitle())
+                .map(ReadBookResponseCreator::create)
+                .switchIfEmpty(Flux.empty());
+
+    }
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/ReadPolyBookReactiveService.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/UpdateBookReactiveService.java [hunk 34]
```diff
@@ -0,0 +1,69 @@
...
+     *
+     * @param book bookEntity to be updated
+     * @param numOfCopies number of copies to be updated
+     * @return if not null the updated BookEntity
+     */
+    private BookEntity updateBook(BookEntity book, int numOfCopies){
+        if (book != null) book.setNumberOfCopies(numOfCopies);
+        return book;
+    }
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/UpdateBookReactiveService.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/helper/ReadBookResponseCreator.java [hunk 35]
```diff
@@ -0,0 +1,40 @@
...
+        ReadBookResponse response = null;
+        if(book != null) {
+            response = new ReadBookResponse();
+            response.setTitle(book.getTitle());
+            response.setAuthor(book.getAuthor());
+            response.setNumberOfCopies(book.getNumberOfCopies());
+        }
+        return response;
+    }
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/service/helper/ReadBookResponseCreator.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/test/java/io/americanexpress/service/book/rest/config/BookTestConfig.java [hunk 36]
```diff
@@ -0,0 +1,25 @@
...
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+/**
+ * {@code BookTestConfig} contains configurations for testing.
+ */
+@Configuration
+@Import(BookConfig.class)
+public class BookTestConfig {
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/test/java/io/americanexpress/service/book/rest/config/BookTestConfig.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/test/java/io/americanexpress/service/book/rest/service/helper/ReadBookResponseCreatorTest.java [hunk 37]
```diff
@@ -0,0 +1,42 @@
...
+}
```
Comment (andCastStatic) [inline service/service-samples/sample-service-rest-mongodb-book/src/test/java/io/americanexpress/service/book/rest/service/helper/ReadBookResponseCreatorTest.java]: EOF line needed

File: service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/config/BookConfig.java (hunk 15 above)
Comment (whoswendy) [reply service/service-samples/sample-service-rest-mongodb-book/src/main/java/io/americanexpress/service/book/rest/config/BookConfig.java]: removed

**Review by guttff — APPROVED** [review]

**Pull request merged**